bakery/src/cli/execute.js

```diff
@@ -79,7 +79,6 @@ const extractLocalImageDetails = imageArg => {
 const input = (dataDir, name) => `--input=${name}=${dataDir}/${name}`
 const output = (dataDir, name) => {
   const outputDir = path.resolve(dataDir, name)
-  if (fs.existsSync(outputDir)) { fs.rmdirSync(outputDir, { recursive: true }) }
```

Comment:
This was removed intentionally and related to [this change](https://github.com/openstax/output-producer-service/pull/371#pullrequestreview-697360328), but after changing the code in the related comment, I will add this back. 

Below was my thought process before this review and change to related above change:
--- 

I discovered with this line, we are unable to use the same input and output with the CLI. if the CLI detects an output with existing name it will remove it and its contents.

This behavior is not consistent with how the concourse pipeline tasks function, because an input can be used as an output in the same task. ([Reference: example 4](https://blog.concourse-ci.org/introduction-to-task-inputs-and-outputs/))